makemessages.sh

```diff
@@ -1,4 +1,14 @@
 #!/usr/bin/env bash
-cmd="./manage.py makemessages --all --no-wrap --no-location --no-obsolete"
+
```

Comment:
```suggestion
set -e
```

If e. g. `makemessages` fails, drop out with error exit code, instead of silently continuing.